main_timer_rate: 1.0
swarm_timer_rate: 30.0

tracking:
  radius: 3.0
  height: 2.0
